CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Adds custom event category for checkbox change events. Any app with it's own tracking logic for checkboxes will need to remove it (PR #729)
```

Comment:
Should we mark this as a breaking change? Like https://github.com/alphagov/govuk_publishing_components/pull/729/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR82